config/locales/en.yml

```diff
@@ -391,570 +408,624 @@ en:
-          - id: "0007"
-            text: "Get help from Nexus NI"
-            href: "https://nexusni.org/domestic-and-sexual-abuse-24-hour-helpline/"
-            show_to_nations:
-              - Northern Ireland
-          - id: "0008"
-            text: "Get help from Live Fear Free"
-            href: "https://gov.wales/live-fear-free"
-            show_to_nations:
-              - Wales
-          - id: "0009"
-            text: "Get help from Women’s Aid Scotland"
-            href: "https://womensaid.scot/"
-            show_to_nations:
-              - Scotland
-          - id: "0010"
-            text: "Get help from Scotland’s forced marriage and domestic abuse helpline"
-            href: "https://sdafmh.org.uk/"
-            show_to_nations:
-              - Scotland
-          - id: "0011"
-            text: "Get help for children (Childline)"
-            href: "https://www.childline.org.uk/"
-          - id: "0012"
-            text: "Get advice for children (Children’s Commissioner for Wales)"
-            href: "https://www.childcomwales.org.uk/coronavirus/"
-            show_to_nations:
-              - Wales
-          - id: "0013"
-            text: "Contact NSPCC if you’re concerned about a child"
-            href: "https://www.nspcc.org.uk/keeping-children-safe/our-services/nspcc-helpline/"
-          - id: "0014"
-            text: "Find helplines if you’re an older person (Older People’s Commissioner for Wales)"
-            href: "https://www.olderpeoplewales.com/en/coronavirus.aspx"
-            show_to_nations:
-              - Wales
-          - id: "0015"
-            text: "Find victim support helplines (this page is in Welsh)"
-            href: "https://www.victimsupport.org.uk/help-and-support/get-help/support-near-you/wales"
-            show_to_nations:
-              - Wales
+        - id: '0001'
+          text: If you’re in immediate danger call 999 and ask for the Police.
+        - id: '0002'
+          text: If you’re in danger and unable to talk on the phone, call 999, then
+            press 55 when prompted.
+        - id: '0003'
+          text: Call the National Domestic Abuse Helpline on <a href="tel:0808 2000
+            247" class="govuk-link">0808 2000 247</a>
+        - id: '0004'
+          text: If you’re a child or young person call Childline on <a href="tel:0800
+            1111" class="govuk-link">0800 1111</a>
+        - id: '0005'
+          text: Find helplines <a href="https://www.gov.uk/government/publications/coronavirus-covid-19-and-domestic-abuse/coronavirus-covid-19-support-for-victims-of-domestic-abuse"
+            class="govuk-link">if you’re a victim of domestic abuse or feel at risk
+            of abuse</a>, or if you are a <a href="https://www.gov.uk/guidance/coronavirus-covid-19-victim-and-witness-services"
+            class="govuk-link">victim or witness of a crime</a> (these pages have
+            no quick escape)
```

Comment:
```suggestion
          text: |
            Find helplines <a href="https://www.gov.uk/government/publications/coronavirus-covid-19-and-domestic-abuse/coronavirus-covid-19-support-for-victims-of-domestic-abuse"
            class="govuk-link">if you’re a victim of domestic abuse or feel at risk
            of abuse</a>, or if you are a <a href="https://www.gov.uk/guidance/coronavirus-covid-19-victim-and-witness-services"
            class="govuk-link">victim or witness of a crime</a> (these pages have
            no quick escape)
```

Reply:
Confirmed renders ok, 
![image](https://user-images.githubusercontent.com/3694062/82885142-a1d27580-9f3c-11ea-8b7f-36b39bf45f65.png)

See here for why i think we shouldn't change these: https://github.com/alphagov/govuk-coronavirus-find-support/pull/222/commits/b0c5b0aafdd33f6b983cf66d96c9d34c0e9bfcf6#r430281216